app/views/graphiql/rails/editors/show.html.erb

```diff
@@ -55,13 +55,22 @@
 
     // Defines a GraphQL fetcher using the fetch API.
     var graphQLEndpoint = "<%= graphql_endpoint_path %>";
+    var graphQLMethod = "<%= graphql_endpoint_method %>";
     function graphQLFetcher(graphQLParams) {
-      return fetch(graphQLEndpoint, {
-        method: 'post',
+      var query = graphQLEndpoint;
+      var params = {
+        method: graphQLMethod,
         headers: { 'Content-Type': 'application/json' },
-        body: JSON.stringify(graphQLParams),
-        credentials: 'include',
-      }).then(function (response) {
+        credentials: 'include'
+      };
+
+      if (graphQLMethod.toLowerCase() === 'get') {
+        query += "?query=" + graphQLParams.query;
```

Comment:
Oh, good point, we need to add the `variables` there as well. What about something like this: 

```javascript
var queryParams = Object.keys(graphQLParams).map(function(key) {
  return key + "=" + graphQLParams[key];
 });

query += "?" + queryParams.join("&");
```

So we always send all the params in `graphQLParams`, like we do for `POST` requests.